portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase

```diff
@@ -65,14 +65,46 @@ definition {
 	}
 
 	@description = "Verify that a Rich Text Field can be edited"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeEdited {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133110 CanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Rich Text");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		FormViewBuilder.clickFieldLabel(fieldLabel = "Rich Text");
+
+		FormViewBuilder.editFieldLabel(
+			fieldLabel = "Rich Text",
+			label = "Test Rich Text");
```

Comment:
Hi @ewertonluna, I just noticed that `FormViewBuilder.clickFieldLabel` macro belongs to AppBuilder, not to Forms. Please, check if `FormsAdminNavigator.gotoEditField` macro works for this case.

The `FormViewBuilder.editFieldLabel` is also from AppBuilder so we need to avoid this, since this folder will be deleted soon.

After some research I saw something that maybe you could try:
```
Click(key_name = "Multiple Selection",
locator1 = "FormFields#FIELD_TARGET_LABEL");

Form.editFieldLabelMetal(fieldValue = "Test Multiple Selection");
```
Please, check if this is applicable for your case.

Thanks.